python/chronos/src/bigdl/chronos/data/__init__.py

```diff
@@ -15,3 +15,4 @@
 #
 
 from .tsdataset import TSDataset
+from .repo_dataset import get_public_dataset, gen_synthetic_data
```

Comment:
change the document as well
https://bigdl.readthedocs.io/en/latest/doc/Chronos/Overview/quick-tour.html
https://bigdl.readthedocs.io/en/latest/doc/Chronos/Overview/data_processing_feature_engineering.html#built-in-dataset

Reply:
After searching in vscode, there exists "from bigdl.chronos.data.repo_dataset import get_public_dataset" in other documents as well, maybe can create a new PR to change all the documents.

Reply:
yep, we will do it in another PR.